.ci/docker/build.sh

```diff
@@ -192,6 +192,21 @@ case "$tag" in
     UCC_COMMIT=${_UCC_COMMIT}
     INDUCTOR_BENCHMARKS=yes
```

Comment:
Works, although could be made more concise by having a condition like: `if [[ $tag =~ "benchmarks" ]];` to avoid repetition